recipes/palikiss/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+make PREFIX=$PREFIX CC=$CC -C Misc/Applications/pAliKiss all
+make PREFIX=$PREFIX CC=$CC -C Misc/Applications/pAliKiss install-program
+make PREFIX=$PREFIX CC=$CC -C Misc/Applications/lib install
```

Comment:
```suggestion
make -j ${CPU_COUNT} PREFIX=$PREFIX CC=$CC -C Misc/Applications/lib install
```

Reply:
same here: no real use in running in parallel as it just copies a few set of files

Reply:
Why does it need ` CC=$CC` ?

Reply:
see above